src/components/views/messages/MImageBody.js

```diff
@@ -454,3 +482,22 @@ export default class MImageBody extends React.Component {
         </span>;
     }
 }
+
+export class HiddenImagePlaceholder extends React.PureComponent {
+    static propTypes = {
+        hover: PropTypes.bool,
+    };
+
+    render() {
+        let className = 'mx_HiddenImagePlaceholder';
+        if (this.props.hover) className += ' mx_HiddenImagePlaceholder_hover';
+        return (
+            <div className={className}>
+                <div className='mx_HiddenImagePlaceholder_button'>
+                    <img src={require("../../../../res/img/feather-customised/eye.svg")} width={17} height={12} />
```

Comment:
This would ideally be a masked css background, like other icons, as `<img>` won't adapt to other accent colors in different themes.

Reply:
Hmm, true. I think the SVG needs some touchups for that to work. Will look into it in a secondary PR.